### DISCUSSION TITLE
AI導致工程師裁員潮

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2069] [General]
https://www.gvm.com.tw/article/110011
https://www.yourator.co/company_blogs/617
過去有聽過一些新聞關於AI工程師完成了人工智慧的開發後就被公司裁員，讓AI取代員工，這也讓我思考工程師是否還有未來以及該如何避免被取代。雖然AI 無法像軟體工程師一樣處理問題的發生，然而AI的出現，降低了程式設計的門檻。最有感的就是這次做的程式設計報告，即使我並沒有豐富學習程式的經歷，但只要利用chatgpt得到大致的程式，再經過細微的修改，讓我克服了第一次的實作。而我認爲避免被取代的關鍵在於文中所說「懂得善用 AI 工具的工程師，不僅能降低工作負擔，也能更有效地完成開發」，因此我們在學習的時候，需要利用AI而不是依靠AI。

**Comment by JoannaLai909** [comment]
我同意你的看法，作為一個對程式設計零基礎的人來說chat gpt在實作上真的幫了我很大的忙，但說實話它寫出來的程式碼很多都是有錯誤的或是不完整的，所以最後還是要花時間去解讀每段程式碼的意思，然後自己修改或補充，不然就是要把丟給chat gpt的問題問的更詳細，並且給予更多細節，才能降低它的錯誤率，因此現階段來說我覺得ai並不能完全取代工程師

**Comment by hunteryang0108** [comment]
我認為工程師的職務沒有辦法以做一個程式專題來類比，商業上的開發行為並不是像做一個學生專題那麼容易。雖然某些類型的工程師確實有可能會被AI所取代，例如：前端工程師等，但這並不代表可以否定其他工程師的開發能力。如演算法工程師、UX工程師等職位，以現在的科技來說，這些工作還是需要依靠人類的智慧的。
